src/usr/local/pkg/miniupnpd.xml

```diff
@@ -26,22 +26,21 @@
 			<description>
 				<![CDATA[
 				Universal Plug and Play Internet Gateway Device (UPnP IGD) and Port Control Protocol (PCP) are
-				network protocols which allow local software and devices to configure each other when attaching
-				to a network. This includes autonomously creating dynamic NAT rules to redirect and pass
-				incoming connections from remote hosts.
+				network protocols which allow local software and devices to autonomously create dynamic NAT
```

Comment:
The current text is fine given that clients may change their configuration as well, e.g. as part of the reported download/upload values

Reply:
I see, thanks. Let's keep the current text in this case.

Reply:
Remove this change.